src/calliope/backend/backend_model.py

```diff
@@ -923,8 +911,8 @@ def _rebuild_references(self, references: set[str]) -> None:
                 self.delete_component(ref, component)
                 getattr(self, "add_" + component.removesuffix("s"))(name=ref)
 
-    def _get_capacity_bound(
-        self, bound: Any, name: str, references: set, fill_na: Optional[float] = None
+    def _get_variable_bound(
```

Comment:
Just for clarity: this is _either_ a lower or upper bound, not both at the same time?
If that's the case, could you change the the docstring to reflect it (`...the upper or lower...`)?

Otherwise, `_bounds` might be better.

Reply:
I see a the latex model uses `_get_variable_bounds`, which is a tad confusing.